src/Components/Publishing/Layouts/NewsLayout.tsx

```diff
@@ -123,7 +123,7 @@ export class NewsLayout extends Component<Props, State> {
     return (
       <>
         {shouldAdRender && (
-          <NewDisplayCanvas
+          <DisplayAd
```

Comment:
In addition to converting this from a `NewDisplayCanvas` to a `DisplayAd` component, I would actually remove the ternary [on line 188](https://github.com/artsy/reaction/blob/94d93024a7874f4e6fff4b7ed78a8fe9e5b861de/src/Components/Publishing/Layouts/NewsLayout.tsx#L188) and return the result of the function call to [renderAd](https://github.com/artsy/reaction/blob/94d93024a7874f4e6fff4b7ed78a8fe9e5b861de/src/Components/Publishing/Layouts/NewsLayout.tsx#L125), but be sure to include the check for `areHostedAdsEnabled` since we only needed that duplicated `CanvasFooter` code to render NewsArticles both with and without ads.